libfuse/fs.go

```diff
@@ -117,28 +120,40 @@ func NewFS(config libkbfs.Config, conn *fuse.Conn, debug bool, platformParams Pl
 		notifications:  libfs.NewFSNotifications(log),
 		platformParams: platformParams,
 		quotaUsage:     libkbfs.NewEventuallyConsistentQuotaUsage(config, "FS"),
+		nextInode:      2, // root is 1
 	}
 	fs.root.private = &FolderList{
 		fs:      fs,
 		tlfType: tlf.Private,
 		folders: make(map[string]*TLF),
+		inode:   fs.assignInode(),
 	}
 	fs.root.public = &FolderList{
 		fs:      fs,
 		tlfType: tlf.Public,
 		folders: make(map[string]*TLF),
+		inode:   fs.assignInode(),
 	}
 	fs.root.team = &FolderList{
 		fs:      fs,
 		tlfType: tlf.SingleTeam,
 		folders: make(map[string]*TLF),
+		inode:   fs.assignInode(),
 	}
 	fs.execAfterDelay = func(d time.Duration, f func()) {
 		time.AfterFunc(d, f)
 	}
 	return fs
 }
 
+func (f *FS) assignInode() uint64 {
+	f.inodeLock.Lock()
+	defer f.inodeLock.Unlock()
+	next := f.nextInode
+	f.nextInode++
```

Comment:
It seems this is simple enough to use `atomic.AddUint64` without introducing mental overhead. But I guess assigning new `inode` isn't that frequent either.